client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -390,7 +390,7 @@ export const submitAddressForNormalization = ( orderId, siteId, group ) => (
 			group
 		];
 
-		if ( isEqual( values, normalized ) ) {
+		if ( isEqualCaseInsensitive( values, normalized ) ) {
```

Comment:
The server returns a `is_trivial_normalization` flag just for these cases, and it's even a bit smarter than this (you can look for the function `checkTrivialNormalization` on the server repo). That flag is already used [here](https://github.com/Automattic/wp-calypso/blob/7b6eb2e9263df5b602240376a5552e1e31c3f074/client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js#L243) to transparently use the normalized values. So, when the normalization is considered "trivial" by the server, the normalized values will automatically override the user values, so the `isEqual()` function will work just fine.

Do you have an example when that logic is not working as it should?